src/name_bazaar/ui/pages/user_offerings_page.cljs

```diff
@@ -56,58 +57,61 @@
         route-params (subscribe [:district0x/route-params])]
     (fn [{:keys [:title :no-items-text]}]
       (let [{:keys [:items :loading? :params :total-count]} @search-results]
-        [app-layout
-         [ui/Segment
-          [ui/Grid
-           {:padded true
-            :class "no-inner-horizontal-padding mobile-inner-vertical-padding"}
-           [ui/GridRow
-            {:class :join-upper}
-            [ui/GridColumn
-             {:computer 8
-              :tablet 8
-              :mobile 16}
-             [:h1.ui.header title]]
-            [ui/GridColumn
-             {:computer 6
-              :tablet 8
-              :mobile 16
-              :floated "right"}
-             (when (:user/address @route-params)
-               [share-buttons
-                {:url
-                 @(subscribe [:page-share-url :route.user/offerings (select-keys @route-params [:user/address])])
-                 :title
-                 (str title " on NameBazaar")}])]]
-           [ui/GridRow
-            {:vertical-align :bottom}
-            [ui/GridColumn
-             {:computer 8
-              :tablet 8
-              :mobile 16}
-             [user-offerings-search-params]]
-            [ui/GridColumn
-             {:computer 6
-              :tablet 8
-              :mobile 16
-              :floated "right"}
-             [user-offerings-order-by-select]]]]
-          [offering-infinite-list
-           {:class "primary"
-            :total-count total-count
-            :offset (:offset params)
-            :loading? loading?
-            :no-items-text no-items-text
-            :on-next-load (fn [offset limit]
-                            (dispatch [:offerings.user-offerings/set-params-and-search
-                                       {:offset offset :limit limit} {:append? true}]))}
-           (doall
-             (for [[i offering] (medley/indexed items)]
-               [offering-list-item
-                {:key i
-                 :offering offering
-                 :header-props {:show-sold? true
-                                :show-missing-ownership? true}}]))]]]))))
+        [:div.wrapper
+         [name-bazaar-meta-tags {:title title :description (if-let [address (:user/address @route-params)]
```

Comment:
:title "NameBazaar - [title]"
:description "See all ENS name offerings of [address]"